components/uikit/Input/PasswordInput.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Input,
+  InputField,
+  InputSlot,
+  InputIcon,
+  FormControl,
+  AlertCircleIcon,
+  FormControlError,
+  FormControlErrorIcon,
+  FormControlErrorText,
+  FormControlHelper,
+  FormControlHelperText,
+} from "@gluestack-ui/themed";
+
+export type PasswordInputProps = {
+    error?: string;
+    help?: string;
+    isRequired?: boolean;
+    icon: React.ElementType;
+    value?: string;
+    onChange?: (value: string) => void;
+  };
+
+export const PasswordInput = ({error, help, isRequired, icon, value, onChange}:PasswordInputProps) => {
+    const [showPassword, setShowPassword] = useState(false)
+    const handleState = () => {
+      setShowPassword((showState) => {
+        return !showState
+      })
+    }
+    return (
+      <Box>
+        <FormControl isRequired={isRequired}>
+          <Input
+            px="$4"
+            h={58}
+            borderRadius="$xl"
+            borderWidth="$1"
+            borderColor="$primaryDark"
+            $focus-borderWidth={1.5}
+          >
+            <InputField
+              type={showPassword ? "text" : "password"}
+              placeholder="Password"
```

Comment:
```suggestion
              placeholder="Mot depasse"
```
Ou ajoutes une prop pour le placeholder